torch/quantization/quantize_fx.py

```diff
@@ -51,14 +67,32 @@ def _prepare_standalone_module_fx(graph_module, qconfig_dict, inplace=False):
                                    custom module is observed or not
 
     """
-    return _prepare_fx(graph_module, qconfig_dict, inplace, is_standalone_module=True)
+    return _prepare_fx(model, qconfig_dict, inplace, is_standalone_module=True)
+
+
+def fuse_fx(model, inplace=False):
+    r""" Fuse modules like conv+bn, conv+bn+relu etc, model must be in eval mode.
+    Fusion rules are defined in torch.quantization.fx.fusion_pattern.py
+    Args:
+        `model`: a torch.nn.Module model
+        `inplace`: flag for whether we fuse modules inplace or out of place
 
-def prepare_fx(graph_module, qconfig_dict, inplace=False):
+    Example:
+    ```python
+    from torch.quantization import fuse_fx
+    m = Model().eval()
+    m = fuse_fx(m)
+    ```
+    """
+    assert not model.training, 'fuse_fx only works on models in eval mode'
```

Comment:
Where is fusion done for QAT?

Reply:
this is the top level fusion function. we have another function `_fuse_fx` which does not check this flag. since fusion for QAT only makes sense for quantization so i didn't expose that